sys/net/include/socket_base/socket.h

```diff
@@ -24,7 +24,8 @@
 
 #include <stdint.h>
 
-#include "net/gnrc/ipv6.h"
+#include "net/af.h"
+#include "net/ipv6/addr.h"
```

Comment:
OT: the `net/ipv6` and `net/ng_ipv6` will get merged to `net/ipv6` soon-ish right?